REFERENCE.md

```diff
@@ -139,13 +139,112 @@ appmap.recordScript(
   "(function main () { return 123; } ());",
   "path/to/main.js",
 );
-const trace = appmap.stop(track, {
+const trace = appmap.stopTrack(track, {
   status: 0,
-  errors: [] 
+  errors: []
 });
 console.log(JSON.stringify(trace, null, 2));
 ```
 
+### `createAppmap(home, configuration, base)`
```

Comment:
Can this be `createAppMap`?

Reply:
Sure, but this will be a breaking change. If you think that is worth it I can do it. If so, I can also rename `startTrack` and `stopTrack` respectively into `startRecording` and `stopRecording`.

Reply:
What would it break? Anything besides our own stuff? If not, now is probably a good time to rename things.

Reply:
The API already existed before this branch and it was publicly documented. I agree that probably not many people rely on it so it is probably okay to can change it. Nonetheless, I would still add a `BREAKING CHANGE` commit.

Reply:
That sounds fine.